composer.json

```diff
@@ -22,7 +22,7 @@
         "symfony/property-access": "~2.3",
         "twig/twig": "*",
         "knplabs/knp-markdown-bundle": "~1.3.2",
-        "egeloen/ckeditor-bundle": "~2.2",
+        "egeloen/ckeditor-bundle": "~3.0",
```

Comment:
I understand i will correct it.

But it adds a lot of complexity in time IMO. I'm not talking about this bundle but about this comportement in general.

In keeping old version with BC, if a day for example we got a 2.8 version, it's possible to see `~2.2|~3.0|~4.2|~5.1`. Can someone argue a lot more with this usage of composer.

Thanks